themes/default/content/challenge/startup-in-a-box.md

```diff
@@ -508,9 +513,410 @@ expect(await page.title()).toBe("Pulumi Challenge");
+    private bucket: gcp.storage.Bucket;
+    private backendBucket: gcp.compute.BackendBucket;
+    private ip: gcp.compute.GlobalAddress;
+    private httpForwardingRule: gcp.compute.GlobalForwardingRule;
+
+    constructor(name: string, args: any, opts?: pulumi.ComponentResourceOptions) {
+        super("pulumi:challenge:CdnWebsite", name, args, opts);
+
+        this.bucket = new gcp.storage.Bucket("mybucket", {
+            location: "US"
+        });
+
+        const bucketIamBinding = new gcp.storage.BucketIAMBinding("bucket-iam-binding", {
+            bucket: this.bucket.name,
+            role: "roles/storage.objectViewer",
+            members: ["allUsers"],
+        });
+
+        const config = new pulumi.Config();
+        const path = config.get("path") || "./website";
+
+        const syncedFolder = new synced_folder.GoogleCloudFolder("synced-folder", {
+            path: path,
+            bucketName: this.bucket.name,
+        });
+
+        // Google Cloud Load Balancer Backend
+        this.backendBucket = new gcp.compute.BackendBucket("backend-bucket", {
+            bucketName: this.bucket.name,
+            enableCdn: true,
+        });
+
+        // CDN Configuration
+        this.ip = new gcp.compute.GlobalAddress("ip", {});
+        const urlMap = new gcp.compute.URLMap("url-map", { defaultService: this.backendBucket.selfLink });
+        const httpProxy = new gcp.compute.TargetHttpProxy("http-proxy", { urlMap: urlMap.selfLink });
+
+        this.httpForwardingRule = new gcp.compute.GlobalForwardingRule("http-forwarding-rule", {
+            ipAddress: this.ip.address,
+            ipProtocol: "TCP",
+            portRange: "80",
+            target: httpProxy.selfLink,
+        });
+
+        // We also need to register all the expected outputs for this
+        // component resource that will get returned by default.
+        this.registerOutputs({
+            bucketName: this.bucket.id,
+            cdnUrl: pulumi.interpolate`http://${this.ip.address}`
+        });
+    }
+
+    get url(): pulumi.Output<string> {
+        return pulumi.interpolate`http://${this.ip.address}`;
+    }
+}
+```
+
+Now we can consume this! Awesome. Back in `pulumi-challenge/index.ts`, we now have this:
+
+```typescript
+// Deploy Website to Google Cloud Storage with CDN
+// Also shows the challenger how to build a ComponentResource.
+import { CdnWebsite } from "./cdn-website";
+
+const website = new CdnWebsite("your-startup", {});
+export const cdnUrl = website.url;
+```
+
+#### Step 6. Adding Another Provider
+
+Now that we have our website being delivered as fast as possible via our `CdnWebsite` component and Cloud CDN, how do we know that what we’ve deployed actually works? We could leverage a fantastic service, such as Checkly, to ensure our website passes some sanity checks.
+
+First, we need to add a new provider:
+
+```shell
+npm install @checkly/pulumi
+
+# API KEY: https://app.checklyhq.com/settings/account/api-keys
```

Comment:
I would call out specifically how to create checkly API key by adding couple of sentence:
```
Please go to https://app.checklyhq.com/settings/account/api-keys to create a new API key. Then copy the key to clipboard.
```